code/index.html

```diff
@@ -1,22 +1,112 @@
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Rampart+One&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Source+Sans+Pro:wght@300&display=swap" rel="stylesheet">
+    <!--Title font: "Averia Serif Libre"-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Averia+Serif+Libre:wght@300&display=swap" rel="stylesheet">
+  <!--Signature font: Nothing You Could Do-->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Nothing+You+Could+Do&display=swap" rel="stylesheet">
+    <title>WWN</title>
+    <link rel="icon" href="./new_logo.png" type="image" sizes="any">
+</head>
+<body>
+  <section class="full-container">
+
+  <header class="header-grid-container">
+    <div class="grid-pic">
+      <img src="./new_logo.png"/>
+    </div>
+    <div class="title-grid">
+      <h1>WORLD WILD NEWS</h1>
+    </div>
+  </header>
+  
+  
+  <nav class="navbar">
+      <a href="#"><p>Home</p></a>
+      <a href="#"><p>About</p></a>
+      <a href="#"><p>Contact</p></a>
+    </nav>
+
+  <div class="headline-section-grid">
+
+        <h2>Wally out on adventures.. AGAIN!</h2>
+
+    <div class="bignews-grid1">
+      <img src="https://metro.co.uk/wp-content/uploads/2021/08/PRC_195374162.jpg?quality=90&strip=all&crop=80px%2C0px%2C2240px%2C1261px&resize=650%2C366&zoom=1" alt="Wally the Walrus">
+    </div>
+
+    <div class="bignews-grid2">
+      <p>Wally the Walrus has been spotted again, this time looking like he should be headed for St Tropez as he sits behind the wheel of a pleasure craft. 
+        The famous sea mammal has got quite a bad reputation recently for sinking boats, 
+        but from the look of these photos he is getting his sea legs now. We would hop in for a ride if Wally was sailing the ship. 
+        Ahoy there sailor!</p>
+    </div>
+
+  </div>
+
+  <div class="body-section-grid">
+
+    <div class="smallnews-grid">
```

Comment:
occasional use of <article> or <section> could be also a nice decision for a semantic markup and better accessibility of the page content.

Reply:
Good point, changed it to article instead, thank you @MarySnopok